WordPress/Classes/ViewRelated/Me/ChangePasswordViewController.swift

```diff
@@ -0,0 +1,73 @@
+class ChangePasswordViewController: SettingsTextViewController, UITextFieldDelegate {
+    typealias ChangePasswordSaveAction = (String) -> ()
+
+    private var onSaveActionPress: ChangePasswordSaveAction?
+    private var currentValue: String = ""
+    private lazy var saveBarButtonItem: UIBarButtonItem = {
+        let saveItem = UIBarButtonItem(title: Constants.actionButtonTitle, style: .plain, target: nil, action: nil)
+        saveItem.on() { [weak self] _ in
+            self?.save()
+        }
+        return saveItem
+    }()
+
+    convenience init(onSaveActionPress: @escaping ChangePasswordSaveAction) {
+        self.init(text: "", placeholder: "\(Constants.title)...", hint: Constants.description)
+        self.onSaveActionPress = onSaveActionPress
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        mode = .password
```

Comment:
This should be `.newPassword`.

I believe to get iOS 12's password generation to work there also has to be a textfield containing the username with `mode = .username`, however that will require some experimenting.

Reply:
New `SettingsTextModes` added: `SettingsTextModesNewPassword`. This will add:
```
if (@available(iOS 12.0, *)) {
   self.textField.textContentType = UITextContentTypeNewPassword;
}
```
Thanks for the hint!